packages/storybook/src/schematics/configuration/root-files/.storybook/webpack.config.js

```diff
@@ -0,0 +1,36 @@
+// Export a function. Accept the base config as the only param.
+module.exports = async ({ config, mode }) => {
```

Comment:
I'm curious, why is an initial `webpack.config.js` needed for loading css?
As a reference, the webpack config of `@storybook/angular`: https://github.com/storybookjs/storybook/blob/next/app/angular/src/server/framework-preset-angular.ts

Reply:
Because that would be limiting us to storybook/angular

Reply:
That's true, haven't thought about this 🙂  

In case you didn't knew there is also a presets API https://storybook.js.org/docs/presets/writing-presets/

It's basically about putting the code out of the schematic/bootstrapped storybook inside an internal preset, lets name it `@nrwl/storybook/preset-angular`, `@nrwl/storybook/preset-react` , etc , and maintain the configuration from the inside of your package instead of updating it via e.g. `ng update` on the end-users side 

Those presets can also be overwritten by the end-user, if needed.
The cons: The user doesn't know what happens behind a preset (which is kind of the angular way of things but not entirely true for e.g. react I guess?)